docs/modules/deployment/pages/core/centos-rhel7/install-core.adoc

```diff
@@ -5,25 +5,25 @@ sudo yum -y install https://yum.opennms.org/repofiles/opennms-repo-stable-rhel7.
 sudo rpm --import https://yum.opennms.org/OPENNMS-GPG-KEY
 ----
 
-.Installation of {page-component-title} with all built-in dependencies
+.Install {page-component-title} with all built-in dependencies (optional)
```

Comment:
@indigo423 , because the TIP below the command says this is optional, I thought it was good to put that in the title before the user runs the command. But I have removed it.

Reply:
The TIP is referencing the following section - I could see how this could be confusing to users.  There might be a few other places with a TIP or other note that precedes the section it refers to.